### ISSUE TITLE
calendar event being created with wrong date on iPhone

### ISSUE THREAD

**Issue by marcusx2** [AnandChowdhary/calendar-link#289]
If I use the `ics` option on Safari iOS, and use the following start date: _2021-03-01 18:00:00_ and end date: _2021-03-01 23:00:00_, it creates an event on the iPhone's native calendar with this date: _2021-01-01 23:08:00_. I'm not having problems with any other calendar. Only on iPhone the event is being created with the wrong date.

**Comment by marcusx2** [comment]
I tried using [this website](https://datebook.dev/generators/) to generate a .ics file with the exact same input, to see if the event would be generated with the wrong date. It wasn't. It created an event with the correct date. I am attaching the .ics file that is generated by `calendar-link` and the one generated by `datebook` so you can compare. Interestingly, the `DTSTART` and `DTEND `are the same for both, but `calendar-link`'s ics file has less fields. My guess is that one or more of the fields that are missing are essential for the iOS to get the date correctly. The order of some fields is also different. That could be the issue as well, although I think it's less likely.

`calendar-link` version
```
BEGIN:VCALENDAR
VERSION:2.0
BEGIN:VEVENT
DTSTART:20210302T210000Z
DTEND:20210303T020000Z
SUMMARY:The Great Event
DESCRIPTION:This is the greatest event ever!
LOCATION:Where the great event will take place, of course!
END:VEVENT
END:VCALENDAR
```

`datebook` version
```
BEGIN:VCALENDAR
VERSION:2.0
BEGIN:VEVENT
CLASS:PUBLIC
DESCRIPTION:This is the greatest event ever!
DTSTART:20210302T210000Z
DTEND:20210303T020000Z
LOCATION:Where the great event will take place, of course!
SUMMARY:The Great Event
TRANSP:TRANSPARENT
END:VEVENT
END:VCALENDAR
UID:pp40o0rvow
DTSTAMP:20210102
PRODID:datebook.dev
```

**Comment by marcusx2** [comment]
No, no, no...that's not the issue. Not exactly. The issue is that the ics file is being generated wrong on iOS. Because I ran it on Chrome for Windows, generated the .ics file and opened it with my iPhone. It created an event with the correct date. But if I run the code on iOS Safari, the event created has a wrong date. 

I guess this makes the problem more complicated? ;_;.

But yeah, both ics files that I posted on the previous comment are correct and will generate the right event on the iPhone.
